tutorials/Machine_Translation_using_PyTorch_Ignite.ipynb

```diff
@@ -0,0 +1,2295 @@
+     "model_name": "DescriptionStyleModel",
+     "model_module_version": "1.5.0",
+     "state": {
+      "_view_name": "StyleView",
+      "_model_name": "DescriptionStyleModel",
+      "description_width": "",
+      "_view_module": "@jupyter-widgets/base",
+      "_model_module_version": "1.5.0",
+      "_view_count": null,
+      "_view_module_version": "1.2.0",
+      "_model_module": "@jupyter-widgets/controls"
+     }
+    },
+    "307baa6d3c3442a386e8b5f9d3eb68a6": {
+     "model_module": "@jupyter-widgets/base",
+     "model_name": "LayoutModel",
+     "model_module_version": "1.2.0",
+     "state": {
+      "_view_name": "LayoutView",
+      "grid_template_rows": null,
+      "right": null,
+      "justify_content": null,
+      "_view_module": "@jupyter-widgets/base",
+      "overflow": null,
+      "_model_module_version": "1.2.0",
+      "_view_count": null,
+      "flex_flow": null,
+      "width": null,
+      "min_width": null,
+      "border": null,
+      "align_items": null,
+      "bottom": null,
+      "_model_module": "@jupyter-widgets/base",
+      "top": null,
+      "grid_column": null,
+      "overflow_y": null,
+      "overflow_x": null,
+      "grid_auto_flow": null,
+      "grid_area": null,
+      "grid_template_columns": null,
+      "flex": null,
+      "_model_name": "LayoutModel",
+      "justify_items": null,
+      "grid_row": null,
+      "max_height": null,
+      "align_content": null,
+      "visibility": null,
+      "align_self": null,
+      "height": null,
+      "min_height": null,
+      "padding": null,
+      "grid_auto_rows": null,
+      "grid_gap": null,
+      "max_width": null,
+      "order": null,
+      "_view_module_version": "1.2.0",
+      "grid_template_areas": null,
+      "object_position": null,
+      "object_fit": null,
+      "grid_auto_columns": null,
+      "margin": null,
+      "display": null,
+      "left": null
+     }
+    }
+   }
+  }
+ },
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4G9j3D6Qk4jV"
+   },
+   "source": [
+    "# Machine Translation using PyTorch Ignite\n",
+    "This tutorial is a brief introduction on how you can train a machine translation model(or any other seq2seq model) using PyTorch Ignite. \n",
+    "\n",
+    "This notebook uses Models, Dataset and Tokenizers from Huggingface, hence they can be easily replaced by other models from the \ud83e\udd17 Hub.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "S5_EFWtFlmms"
+   },
+   "source": [
+    "Installing Essential Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "id": "OUsi2Pv957NV"
+   },
+   "source": [
+    "# installing appropriate modules\n",
+    "%%capture\n",
+    "!pip install git+https://github.com/pytorch/ignite.git@master;\n",
+    "!pip install git+https://github.com/huggingface/transformers.git@master;\n",
+    "!pip install git+https://github.com/huggingface/datasets.git@master;\n"
```

Comment:
```suggestion
    "!pip install git+https://github.com/huggingface/datasets.git@master"
```
Please remove empty lines at the end of each code cell.

Reply:
This was done by running [jblack](https://github.com/irahorecka/jupyterblack). I have removed it for now but having a black formatting for all examples might be good!